tests/scene/test_style_box_texture.h

```diff
@@ -0,0 +1,194 @@
+		style_box_texture->set_expand_margin_individual(3, 4, 5, 6);
+
+		CHECK(style_box_texture->get_expand_margin(SIDE_LEFT) == 3);
+		CHECK(style_box_texture->get_expand_margin(SIDE_TOP) == 4);
+		CHECK(style_box_texture->get_expand_margin(SIDE_RIGHT) == 5);
+		CHECK(style_box_texture->get_expand_margin(SIDE_BOTTOM) == 6);
+	}
+}
+
+TEST_CASE("[StyleBoxTexture] set_region_rect, get_region_rect") {
+	Ref<StyleBoxTexture> style_box_texture = memnew(StyleBoxTexture);
+
+	style_box_texture->set_region_rect(Rect2(1, 1, 1, 1));
+	CHECK(style_box_texture->get_region_rect() == Rect2(1, 1, 1, 1));
+}
+
+TEST_CASE("[StyleBoxTexture] set_draw_center, get_draw_center") {
+	Ref<StyleBoxTexture> style_box_texture = memnew(StyleBoxTexture);
+
+	style_box_texture->set_draw_center(false);
+	CHECK(style_box_texture->is_draw_center_enabled() == false);
+}
+
+TEST_CASE("[StyleBoxTexture] set_h_axis_stretch_mode, set_v_axis_stretch_mode, get_h_axis_stretch_mode, get_v_axis_stretch_mode") {
+	Ref<StyleBoxTexture> style_box_texture = memnew(StyleBoxTexture);
+
+	SUBCASE("set_h_axis_stretch_mode, get_h_axis_stretch_mode") {
+		style_box_texture->set_h_axis_stretch_mode(style_box_texture->AXIS_STRETCH_MODE_TILE);
+		CHECK(style_box_texture->get_h_axis_stretch_mode() == style_box_texture->AXIS_STRETCH_MODE_TILE);
+
+		style_box_texture->set_h_axis_stretch_mode(style_box_texture->AXIS_STRETCH_MODE_TILE_FIT);
+		CHECK(style_box_texture->get_h_axis_stretch_mode() == style_box_texture->AXIS_STRETCH_MODE_TILE_FIT);
+
+		style_box_texture->set_h_axis_stretch_mode(style_box_texture->AXIS_STRETCH_MODE_STRETCH);
+		CHECK(style_box_texture->get_h_axis_stretch_mode() == style_box_texture->AXIS_STRETCH_MODE_STRETCH);
+	}
+
+	SUBCASE("set_v_axis_stretch_mode, get_v_axis_stretch_mode") {
+		style_box_texture->set_v_axis_stretch_mode(style_box_texture->AXIS_STRETCH_MODE_TILE);
+		CHECK(style_box_texture->get_v_axis_stretch_mode() == style_box_texture->AXIS_STRETCH_MODE_TILE);
+
+		style_box_texture->set_v_axis_stretch_mode(style_box_texture->AXIS_STRETCH_MODE_TILE_FIT);
+		CHECK(style_box_texture->get_v_axis_stretch_mode() == style_box_texture->AXIS_STRETCH_MODE_TILE_FIT);
+
+		style_box_texture->set_v_axis_stretch_mode(style_box_texture->AXIS_STRETCH_MODE_STRETCH);
+		CHECK(style_box_texture->get_v_axis_stretch_mode() == style_box_texture->AXIS_STRETCH_MODE_STRETCH);
+	}
+}
+
+TEST_CASE("[StyleBoxTexture] set_modulate, get_modulate") {
+	Ref<StyleBoxTexture> style_box_texture = memnew(StyleBoxTexture);
+
+	style_box_texture->set_modulate(Color(0, 0, 0, 0));
+	CHECK(style_box_texture->get_modulate() == Color(0, 0, 0, 0));
+}
+
+TEST_CASE("[StyleBoxTexture] get_draw_rect") {
+	Ref<StyleBoxTexture> style_box_texture = memnew(StyleBoxTexture);
+
+	style_box_texture->set_expand_margin_all(5);
+	CHECK(style_box_texture->get_draw_rect(Rect2(0, 0, 1, 1)) == Rect2(-5, -5, 11, 11));
+}
+
+} // namespace TestStyleBoxTexture
+
+#endif // TEST_STYLE_BOX_TEXTURE_H
```

Comment:
Add newline at end of file:

```suggestion
#endif // TEST_STYLE_BOX_TEXTURE_H

```